JavaScript Timer/Readme.md

```diff
@@ -0,0 +1,130 @@
+
+  
+
+<h3  align="center">Javascript Timer</h3>
+
+  
+
+<p  align="center">
+A simple web based Timer built using HTML, CSS and Javascript.
+</p>
+
+<br>
+
+
+<details>
+  <summary><b>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+      <ul>
+        <li><a href="#built-with">Built With</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+   </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+     </li>
+    <li><a href="#contact">Contact</a></li>
+  
+  </ol>
+</details>
+
+  
+
+## About The Project
+
+A simple web timer built using HTML, CSS and Javascript. The timer can be paused and can also be reset.
+
+<p  align="right">(<a  href="#top">back to top</a>)</p>
+
+<hr>
+
+  
+
+### Built With
+
+  
+
+* [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML)
+
+* [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS)
+
+* [JavaScript](https://www.javascript.com/)
+
+  
+
+<p  align="right">(<a  href="#top">back to top</a>)</p>
+
+  
+
+## Getting Started
+
+### Prerequisites
+
+No prerequisites required for adding and executing this chrome extension.
+
+### Installation
+
+  
+
+1. Clone the repository:
+
+```sh
+
+git clone https://github.com/khushi-purwar/WebDev-ProjectKart.git
+
+```
+
+2. Navigate to `JavaScript Timer` Project.
+
+3. Open the `index.html` HTML file with live server.
+
+  
+  
+
+<p  align="right">(<a  href="#top">back to top</a>)</p>
+
+  
+
+## Usage
+
+<div  align="center">
+<img  src="https://raw.githubusercontent.com/LiQuiD-404/WebDev-ProjectKart/master/JavaScript%20Timer/snips/snip1.png"  alt="screenshot" >
+ <br> <br>
+
+</div>
+The timer will start once the green play button is clicked.
+<br>
+
+
+
+<br><br>
+
+<div  align="center">
+<img  src="https://raw.githubusercontent.com/LiQuiD-404/WebDev-ProjectKart/master/JavaScript%20Timer/snips/snip2.png"  alt="screenshot" >
+</div>
+
+The green play button will now change into a pause button which can be used to pause the timer. The reset button (beside the pause button) can be used to reset the timer.
+
+  
+  
+
+<p  align="right">(<a  href="#top">back to top</a>)</p>
+
+  
+  
+  
+  
+  
+  
+
+## Contact
+
+Shubham Singh - [LinkedIn](https://www.linkedin.com/in/shubham-singh-519769220/)
+
+<p  align="right">(<a  href="#top">back to top</a>)</p>
```

Comment:
Adding personal socials is not a good practice for any open-source

Reply:
Oh, I am extremely sorry. I will fix this issue right away.